package/oxide/package

```diff
@@ -2,24 +2,20 @@
 pkgname=oxide
 pkgdesc="Launcher application"
 url=https://github.com/Eeems/oxide
-pkgver=2.0~beta~1
+pkgver=2.0~beta~2
```

Comment:
The `pkgrel` part of the package version should be separated from the rest using a dash, not a tilde, i.e. `2.0~beta-2` instead of `2.0~beta~2`. Otherwise, if a version is released that drops the `~beta` suffix, we get `2.0~2`, which is actually sorted lower than `2.0~beta~2`. You can use `opkg compare-versions` to test if a version is sorted lower or higher than another.